proxy/helpers_test.go

```diff
@@ -0,0 +1,31 @@
+package proxy
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMarshalRequest(t *testing.T) {
+	request, err := marshalRequest(1, []string{"0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675"})
+
+	type Request struct {
```

Comment:
A test file should not define a new structure type. It should instead reference a type defined somewhere in the code (which we are testing against)

Reply:
No need for this file anymore due to the above.